mscore/palettebox.cpp

```diff
@@ -552,6 +553,15 @@ bool PaletteBox::eventFilter(QObject* obj, QEvent *event)
                   if (keyEvent->key() == Qt::Key_Up || keyEvent->key() == Qt::Key_Down ||
                           keyEvent->key() == Qt::Key_Enter || keyEvent->key() == Qt::Key_Return) {
                         navigation(keyEvent);
+                        for (Palette* p : palettes()) {
```

Comment:
If I understand correctly this is a bit hacky and it uses the searchbox as a proxy to the accessibility system right?
If it's the case, then please add a comment to explain what you are doing here.

Reply:
Yes, it uses the search box as a proxy to the accessibility system. I implemented it this way because while navigating the search results, the keyboard focus never leaves the search box, so I thought the description of the search box could be set to the currently highlighted palette cell name. Also, since the palette cells aren't represented by widgets, it's not possible to set an accessible name and description for each palette cell. Using a widget for each palette cell could change the way palettes are drawn, so I've left the palette cells as is, for now.